src/applications/personalization/dashboard-2/components/Dashboard.jsx

```diff
@@ -145,12 +181,21 @@ const Dashboard = ({
   );
 };
 
+const isClaimsAvailableSelector = createIsServiceAvailableSelector(
```

Comment:
Is this the best place for these selectors to live?

Reply:
Perhaps not, I _just_ added this quickly, copying [how we are already doing it in the ClaimsAndAppeals component](https://github.com/department-of-veterans-affairs/vets-website/blob/8be90e8bad321a67ce4e593111cb0f0d4e4b822a/src/applications/personalization/dashboard-2/components/claims-and-appeals/ClaimsAndAppeals.jsx), which also might not be doing it in the best place? Would you want to move it to live _in_ `mapStateToProps`? Or someplace else?